Amz.py

```diff
@@ -42,6 +41,39 @@ def pause_small():
     time.sleep(1)
 
 
+def give_away_type(driver):
```

Comment:
A few more comments in here plz

Reply:
Added more comments to explain whats going on.